devinfo/src/mountinfo/mod.rs

```diff
@@ -17,8 +17,11 @@ use std::{
     sync::OnceLock,
 };
 
-mod bytebuf;
-mod error;
+/// Contains a type which is Sized and implements io::Read.
+/// Useful as a Read-able alternative to a Vec<u8>.
+pub mod bytebuf;
```

Comment:
Where are we using this?

Reply:
I'd say keep all private unless there's a specific reason to open up

Reply:
Made this module private.